src/twyn/config/config_handler.py

```diff
@@ -119,36 +120,19 @@ def _write_config(self, toml: TOMLDocument, config: ReadTwynConfiguration) -> No
         if "twyn" not in toml["tool"]:  # type: ignore[operator]
             toml["tool"]["twyn"] = {}  # type: ignore[index]
         toml["tool"]["twyn"] = twyn_toml_data  # type: ignore[index]
+
         self._write_toml(toml)
 
+    def _write_toml(self, toml: TOMLDocument) -> None:
```

Comment:
I did not want to create a TomlHandler or something similar (at least not yet), as I don't want to start building things that will add more complexity for a little benefit at this point.

Reply:
Agreed!